pkg/dashboard/server.go

```diff
@@ -35,10 +35,11 @@ func (s Server) StartServer() (string, utils.ControlChan) {
 		log.Errorf("Failed to check that Helm is operational, cannot continue. The error was: %s", err)
 		os.Exit(1) // TODO: propagate error instead?
 	}
-
+	isDevModeWithAnalytics := s.Version == "0.0.0" && os.Getenv("enableDevAnalytics") == "true"
```

Comment:
Env variable should be all-caps and have `HD_` prefix, it's a sort of project standard.
The `s.Version == "0.0.0"` condition is not needed, the second part of condition handles all cases perfectly.